pom.xml

```diff
@@ -18,23 +18,11 @@
             <artifactId>junit</artifactId>
             <version>3.8.2</version>
         </dependency>
-        <dependency>
-            <groupId>org.bouncycastle</groupId>
-            <artifactId>bcprov-jdk15on</artifactId>
-            <version>1.70</version>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.bouncycastle</groupId>
-            <artifactId>bcprov-jdk15on</artifactId>
-            <version>1.70</version>
-            <scope>compile</scope>
-        </dependency>
     </dependencies>
     <properties>
-        <maven.compiler.source>16</maven.compiler.source>
-        <maven.compiler.target>16</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Java 11 was chosen due to reasons:
1.  Java 11 is a stable long-supported version, it can be used easily on all modern systems (Debian, Ubuntu, etc)
2.  Java 11 is used in our projects, that depends on bee2j. It is simpler to support Java 11 in master branch than update two parallel branches for 11 and 16.

Reply:
Ok